minecraft/dial.go

```diff
@@ -348,20 +360,12 @@ func listenConn(conn *Conn, readyForLogin, connected chan struct{}, cancel conte
 	}
 }
 
-func getXBLToken(ctx context.Context, dialer Dialer) (*auth.XBLToken, error) {
-	if dialer.XBLToken != nil {
-		return dialer.XBLToken, nil
+func getSession(ctx context.Context, dialer Dialer) (*session.Session, error) {
```

Comment:
Unclear function naming. Would clarify that this is about an auth session. In the `minecraft` package, `session` is more likely to refer to a connection.